contribute/README.md

```diff
@@ -10,9 +10,13 @@ Thanks for enhancing OpenFisca anyway!
 
 ## Why contribute to OpenFisca?
 
-OpenFisca is a project being developed under the GPLv3 license or later.
+OpenFisca is a project being developed under the AGPL-3.0 license.
 The source code is freely available and modifiable.
 
+We ask for improvements and evolution of OpenFisca to be shared back, for it allows the community to grow through improvements to the _core engine_ and updates to the _modelled tax-benefit systems_ (country packages, extensions, reforms, etc.).
```

Comment:
Move added text below old one? (`We encourage users` before `We ask`)

Reply:
I would put the updates to the _modelled tax-benefit systems_ first as it is more common than improvement to the core engine.
Proposition : 
> If you improve or update _modelled tax-benefit systems_ (country packages, extensions, reforms, etc.) or the  _core engine_, We ask that you share them back, for it allows the community to grow :)

Reply:
Done.